src/fast_pbkdf2/rebar.config

```diff
@@ -0,0 +1,58 @@
+{erl_opts, [
+  deterministic
+ ]}.
+
+{deps, []}.
+
+{profiles, [
+  {test, [
+    {erl_opts, []},
+    {deps, [
+      {base16, "2.0.1"},
+      {proper, "1.4.0"}
+     ]},
+    {plugins, [
+       {rebar3_codecov, "0.3.0"}
+     ]},
+    {port_env,
+     [
+      {"CFLAGS", "$CFLAGS -std=c99 -O3 -g -Wall -Wextra -fPIC --coverage"},
+      {"LDFLAGS", "$LDFLAGS --coverage"},
+      {"LDLIBS", "$LDLIBS -lcrypto"},
+      {"DRV_LINK_TEMPLATE", "$DRV_LINK_TEMPLATE $LDLIBS"}
+     ]}
+   ]}
+ ]
+}.
+
+{plugins, [pc, rebar3_hex]}.
+
+{artifacts, ["priv/fast_pbkdf2.so"]}.
+
+{port_specs,
+ [
+    {
+        % Any arch
+        ".*",
+        % Create library
+        "priv/fast_pbkdf2.so",
+        % From files
+        ["c_src/*.c"],
+        % Using options
+        [ {env, [{"CFLAGS", "$CFLAGS -std=c99 -O3 -g -Wall -Wextra -fPIC"},
```

Comment:
A full CI Jenkins showed that all the architecture and OSes looked good except macos. I think it may just need the openssl dev library installed:

https://ci-couchdb.apache.org/blue/organizations/jenkins/jenkins-cm1%2FFullPlatformMatrix/detail/jenkins-decouple_offline_hash_strength_from_online/1/pipeline/82/

```
[2023-11-07T22:26:54.619Z] Compiling /Users/jenkins2/workspace/ffline_hash_strength_from_online/macos/build/src/fast_pbkdf2/c_src/fast_pbkdf2.c
[2023-11-07T22:26:54.619Z] /Users/jenkins2/workspace/ffline_hash_strength_from_online/macos/build/src/fast_pbkdf2/c_src/fast_pbkdf2.c:35:10: fatal error: 'openssl/sha.h' file not found
[2023-11-07T22:26:54.619Z] #include <openssl/sha.h>
[2023-11-07T22:26:54.619Z]          ^~~~~~~~~~~~~~~
[2023-11-07T22:26:54.619Z] 1 error generated.
[2023-11-07T22:26:54.619Z] ERROR: compile failed while processing /Users/jenkins2/workspace/ffline_hash_strength_from_online/macos/build/src/fast_pbkdf2: rebar_abort
[2023-11-07T22:26:54.619Z] make: *** [couch] Error 1
```